test/upsert/rocksdb-cleanup/drop-cluster-cascade.td

```diff
@@ -0,0 +1,35 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+$ kafka-create-topic topic=upsert
+
+> CREATE CONNECTION conn
+  FOR KAFKA BROKER '${testdrive.kafka-addr}'
+
+> DROP CLUSTER IF EXISTS c1 CASCADE;
+
+> CREATE CLUSTER c1 REPLICAS (replica1 (SIZE '1'));
+
+> CREATE SOURCE upsert
+  IN CLUSTER c1
+  FROM KAFKA CONNECTION conn (TOPIC
+  'testdrive-upsert-${testdrive.seed}'
+  )
+  KEY FORMAT TEXT VALUE FORMAT TEXT
+  ENVELOPE UPSERT
```

Comment:
`WITH (DISK)` right?

Reply:
The default Materialized has the flag `upsert_source_disk_default` to true. So, it's still rocksdb. https://github.com/MaterializeInc/materialize/blob/ec9206e42355bb86bd64f829b82840c3962d645d/test/upsert/mzcompose.py#L37 

I can add the WITH DISK to make it explicit here.

Reply:
Oh I missed that, sorry! Leave it as is!